rust/agama-lib/src/network/client.rs

```diff
@@ -22,6 +22,7 @@ impl<'a> NetworkClient<'a> {
         })
     }
 
+    // It returns an array of NetworkConnection using the connections_proxy
```

Comment:
```suggestion
    // Returns an array of network connections
```

Note: Whether it uses a proxy, is an implementation detail.